### PR TITLE
#163519105 Add pagination for articles

### PR DESCRIPTION
**Pull request by mcaleb808**
#### What does this PR do?
Adds pagination for articles

#### Description of Task to be completed?
- adds pagination components
- adds pagination functionality for articles

#### How should this be manually tested?
- git clone
- npm install
- npm run start
- View articles `base_url/articles`

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#163519105 
#### Screenshots (if appropriate)

image

image

#### Questions:
N/A

### CODE REVIEW

File: src/_styles/components/_Toast.scss [hunk 1]
```diff
@@ -0,0 +1,179 @@
+.toast {
+  display: flex;
+  align-items: center;
+  visibility: hidden;
+  max-width: 50px;
+  height: 50px;
+  /*margin-left: -125px;*/
```
Comment (NkFab) [inline src/_styles/components/_Toast.scss]: @Caleb remove this comment

Comment (mcaleb808) [reply src/_styles/components/_Toast.scss]: @NkFab Thanks for spotting this, I removed it!

**Review by NkFab — APPROVED** [review]

**Review by filschristian — APPROVED** [review]

File: src/components/common/Pagination.jsx [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+        />
+      );
+      return next;
+    }
+  };
+
+  onPrev = () => {
+    const { currentPage, totalPages } = this.props;
+    if (currentPage > 1 && currentPage <= totalPages) {
+      const next = (
```
Comment (kimotho-njoki) [inline src/components/common/Pagination.jsx]: why cant we directly return this?

Comment (mcaleb808) [reply src/components/common/Pagination.jsx]: Yes, we can! I refactored the code

**Pull request merged**